Slider/Assets/Scripts/UI/Artifact/Screens/ArtifactScreenAnimator.cs

```diff
@@ -142,8 +143,34 @@ private void SetScreensActive(bool value)
         }
     }
 
-    private void OnScreenChange(int index)
+    private void OnScreenChange(int index, int prevIndex)
     {
+        //inventory
+        if(index == 1)
+        {
+            if(prevIndex == 0)
+            {
+                ArtifactInventoryCollectible s = inventory.GetLeftmostSelectible();
+                if(s != null)
+                {
+                    s.controllerSelectible.Select();
+                    s.controllerSelectionImage.enabled = true;
+                    s.UpdateInventoryName();
+                }
```

Comment:
can we just compress this to one method inventory.TrySelectLeftMost() or something